src/queries/user.ts

```diff
@@ -12,6 +13,19 @@ export const orderDetailsByTokenQuery = gql`
   }
 `;
 
+export const userOrderDetailsByTokenQuery = gql`
```

Comment:
WDYT about moving order related queries to separate file?

Reply:
very good idea

Reply:
changed